coordinated-vulnerability-disclosure/index.md

```diff
@@ -0,0 +1,88 @@
+# Coordinated Vulnerability Disclosure (CVD)
+
+EXPERIMENT BY IGDC-DHCW - DRAFT - WORK IN PROGRESS - REQUEST FOR COMMENTS
+
+## Situation - Context and Problem Statement
+
+The upcoming IGDC-DHCW emergency department module (EDM) and its exploratory authentication service (WAuth) are work in progress. Currently these don't seem to have a process for coordinated vulnerability disclosure (CVD), such as if/how any person could report a potential security vulnerability.
+
+The purpose of this document is to open up discussion of a CVD process for the EDM and WAuth services. Then ideally to implement it for those services. Then ideally to extend the CVD to additional IGDC-DHCW services.
+
+The existing guidance from NHS Wales info sec that Joel has found (thus far) doesn't seem to say anything about how people will do security disclosure.
+
+
+## Background - Decision Drivers (Optional)
+
+When a security vulnerability is discovered in a service, it is important to coordinate the disclosure of that vulnerability to the service's users. This is to ensure that the vulnerability is fixed as soon as possible, and that the service's users are not exposed to any risk.
+
+Many popular websites handle this by having a "security" notice on their web page footers, and also by publishing a security process such as how to report a bug, how to report a potential vulnerability, etc. These notices help service providers work better with the public (and internal staff) to handle security issues effectively.
+
+A "Coordinated Vulnerability Disclosure" (CVD) is a process where security vulnerabilities are disclosed to the public only after the responsible parties (like vendors) have had sufficient time to address and mitigate the issue, ensuring a coordinated and timely approach to patching and remediation. 
+
+Background reading:
```

Comment:
Is there an NHS England policy that's accessible?

Reply:
I'll add now. TLDR:

NHS England has a security policy that is derived from UK NCSC as above. Joel's skim didn't turn up anything notable or differentiating.

However, NHS England has multiple weaknesses as compared to NCSC:

  * NHS England doesn't publish a PGP key in the file `security.txt`. 

  * NHS England redirects the reporter to HackerOne, which is a for-profit service, and a blocker, and a security risk, and a cross-border political problem.